### ISSUE TITLE
groundOverlay.setImage() doesn't work properly on Android

### ISSUE THREAD

**Issue by barhbie** [mapsplugin/cordova-plugin-googlemaps#1834]
**I'm submitting a ...**  (check one with "x")
[ ] question
[x ] any problem or bug report
[ ] feature request

**The plugin version:**  (check one with "x")
[ ] **2.0-beta3 (github)** 
[x ] **2.0 (npm)** 

**If you choose 'problem or bug report', please select OS:**  (check one with "x")
[x ] **Android**
[ ] **iOS**

**cordova information:**  (run `$> cordova plugin list`) 

```
com.xmartlabs.cordova.market 1.1 "Market"
cordova-plugin-app-version 0.1.9 "AppVersion"
cordova-plugin-compat 1.1.0 "Compat"
cordova-plugin-console 1.0.7 "Console"
cordova-plugin-device 1.1.6 "Device"
cordova-plugin-dialogs 1.3.3 "Notification"
cordova-plugin-file 4.3.3 "File"
cordova-plugin-file-transfer 1.6.3 "File Transfer"
cordova-plugin-geolocation 2.4.3 "Geolocation"
cordova-plugin-google-analytics 1.8.3 "Google Universal Analytics Plugin"
cordova-plugin-googlemaps 2.0.11 "cordova-plugin-googlemaps"
cordova-plugin-ionic-webview 1.1.15 "cordova-plugin-ionic-webview"
cordova-plugin-screen-orientation 2.0.1 "Screen Orientation"
cordova-plugin-splashscreen 4.0.3 "Splashscreen"
cordova-plugin-statusbar 2.2.3 "StatusBar"
cordova-plugin-whitelist 1.3.2 "Whitelist"
es6-promise-plugin 4.1.0 "Promise"
ionic-plugin-keyboard 2.2.1 "Keyboard"
```

**If you use `@ionic-native/google-maps`, please show me the package.json**
Here's the important part:
```
"dependencies": {
    "@angular/common": "4.0.0",
    "@angular/compiler": "4.0.0",
    "@angular/compiler-cli": "4.0.0",
    "@angular/core": "4.0.0",
    "@angular/forms": "4.0.0",
    "@angular/http": "4.0.0",
    "@angular/platform-browser": "4.0.0",
    "@angular/platform-browser-dynamic": "4.0.0",
    "@ionic-native/app-version": "^3.12.1",
    "@ionic-native/core": "3.6.1",
    "@ionic-native/dialogs": "3.6.1",
    "@ionic-native/file": "^3.12.1",
    "@ionic-native/geolocation": "3.6.1",
    "@ionic-native/google-analytics": "^3.12.1",
    "@ionic-native/google-maps": "4.2.1",
    "@ionic-native/market": "^3.12.1",
    "@ionic-native/screen-orientation": "3.6.1",
    "@ionic-native/splash-screen": "3.4.2",
    "@ionic-native/status-bar": "3.4.2",
    "@ionic-native/transfer": "3.7.0",
    "@ionic/storage": "2.0.1",
    "chart.js": "^2.5.0",
    "com.xmartlabs.cordova.market": "~1.1",
    "cordova-plugin-app-version": "~0.1.9",
    "cordova-plugin-console": "~1.0.5",
    "cordova-plugin-device": "~1.1.4",
    "cordova-plugin-dialogs": "~1.3.3",
    "cordova-plugin-file": "~4.3.3",
    "cordova-plugin-file-transfer": "~1.6.3",
    "cordova-plugin-geolocation": "~2.4.3",
    "cordova-plugin-google-analytics": "~1.8.2",
    "cordova-plugin-googlemaps": "^2.0.11",
    "cordova-plugin-ionic-webview": "~1.1.15",
    "cordova-plugin-screen-orientation": "~2.0.1",
    "cordova-plugin-splashscreen": "~4.0.1",
    "cordova-plugin-statusbar": "~2.2.1",
    "cordova-plugin-whitelist": "~1.3.1",
    "ionic-angular": "3.0.1",
    "ionic-plugin-keyboard": "~2.2.1",
    "ionicons": "3.0.0",
    "moment-timezone": "^0.5.13",
    "rxjs": "5.1.1",
    "sw-toolbox": "3.4.0",
    "zone.js": "^0.8.4"
  }
```

**Current behavior:**
The method setImage() only works once on Android, if you use it multiple times the new image is rendered on top of the old one.

**Expected behavior:**
When using setImage() multiple times, the image of the groundoverlay should change completely and properly, as it does on iOS.

**Steps to reproduce:**
For example: using the following code-example:
https://github.com/mapsplugin/cordova-plugin-googlemaps-doc/blob/master/v2.0.0/class/GroundOverlay/setImage/README.md
But change onClick() to go back and forth between two images (preferably images with transparency to clearly see the bug in effect). The images will be rendered on top of eachother after setImage() is called for a second time.

**Comment by wf9a5m75** [comment]
I guess your application name contains space characters.
If not, please share your project files on github.

**Comment by barhbie** [comment]
I have set up an example project which demonstrates the bug in question here: https://github.com/barhbie/setImageExample

When you click on the "next image"-button the first time, "Image 1" is switched to "Image 2" correctly, but the next time the button is clicked "Image 3" is rendered on top of "Image 2" instead of switching it out (and the next time the next image is rendered on top of both of these two and so on).

In short, clicking "Next image" a few times should make the problem quite clear.

**Comment by wf9a5m75** [comment]
Thank you for creating the demo project. I fixed the bug.
(I haven't checked iOS yet)

Please reinstall the plugin from the multiple_maps branch.

```
$> cordova plugin rm cordova-plugin-googlemaps

$> cordova plugin add https://github.com/mapsplugin/cordova-plugin-googlemaps#multiple_maps --variable ... --variable ... --no-fetch
```

![1834](https://user-images.githubusercontent.com/167831/31975747-2e70bcc4-b8e8-11e7-8462-17abfd810bf9.gif)

**Comment by barhbie** [comment]
Awesome! Just tried it and it works perfectly. Thank you!

**Comment by wf9a5m75** [comment]
I appreciate if you support for this project.

[This is $5 donation link at paypal](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=SQPLZJ672HJ9N&lc=US&item_name=Cordova%2dGoogleMaps%2dPlugin&amount=5%2e00&currency_code=USD&bn=PP%2dDonationsBF%3abtn_donate_LG%2egif%3aNonHosted)
When you donate some amount to the project, please include your issue number in your name.

**Comment by wf9a5m75** [comment]
Thank you for your donation!!!
